vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/cwt/claims/CwtTimeToLiveClaim.kt

```diff
@@ -0,0 +1,37 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList.cwt.claims
+
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.PositiveDuration
+import at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives.PositiveDurationSecondsULongSerializer
+import kotlinx.datetime.Instant
+import kotlinx.serialization.Serializable
+import kotlin.jvm.JvmInline
+import kotlin.time.Duration
+
+/**
+ * 65534 (time to live): OPTIONAL. Unsigned integer (Major Type 0). The time to live claim, if
+ * present, MUST specify the maximum amount of time, in seconds, that the Status List Token can be
+ * cached by a consumer before a fresh copy SHOULD be retrieved. The value of the claim MUST be a
+ * positive number.
+ */
+@Serializable
+@JvmInline
+value class CwtTimeToLiveClaim(
+    @Serializable(with = PositiveDurationSecondsULongSerializer::class)
+    val positiveDuration: PositiveDuration,
+) {
+    data object Specification {
+        const val CLAIM_NAME = "ttl"
+        const val CLAIM_KEY = 65534L
+    }
+
+    fun isInvalid(
```

Comment:
This method is never called

Reply:
fixed in 93ce860696aa4077acb9a053a8b3eac9bee2733b

Well, at least a test has been provided